CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 - Corrected grammer issues and clarified comments in all existing Ruby-based Windows Check and Metric files.
 - Moved existing Ruby-based Windows Check and Metric files to bin/ruby.
 - Changed file naming to the singular instead of the difference in plurality between 'check' and 'metrics'.
+- Dependency on Powersehell version 3.0 or above for powershell checks.
```

Comment:
Only way this is not a breaking change is if all of them actually do require PSv3 which I'd rather play it safe and be wrongly major revving it than accidentally break a user that previously existed with a minor or patch.